### ISSUE TITLE
Carbon density

### ISSUE THREAD

**Issue by Miradin** [pkienzle/periodictable#28]
Hello,
Could you please, point me to the source of the nominal densities you are using in database? Maybe I found one discrepancy in the value for Carbon:
NIST value:  2.26 g/cm^3 [(link)](https://www.physics.nist.gov/cgi-bin/ffast/ffast.pl?Z=6&Formula=&gtype=4&range=S&lower=5&upper=8.05&density=)

Berkeley lab (lbl) value: 2.20 g/cm^3 (from this [page with form](http://henke.lbl.gov/optical_constants/pert_form.html))

Royal Society of Chemistry (RSC) value: 2.20 g/cm^3 [(link)](http://www.rsc.org/periodic-table/element/6/carbon) for graphite.

periodictable value: 
```python
print(periodictable.C.density, periodictable.C.density_units)
2.1 g/cm^3
```

With best regards,
Alex

**Comment by pkienzle** [comment]
The values for density come from the [Neutron Data Booklet](https://www.ill.eu/fileadmin/user_upload/ILL/1_About_ILL/Documentation/NeutronDataBooklet.pdf), which gives a range of density for graphite as 1.9 to 2.3.  The 2.1 value is in the middle of that.

The Neutron Data Booklet copied the data from the LBL X-ray Data Booklet, which cites the CRC (80th edition) for most of its data.  Checking the [100th edition](http://hbcponline.com/faces/documents/04_01/04_01_0001.xhtml) gives the same 2.2 value that you have retrieved from RSC and LBL.  I don't know where they got the range 1.9-2.3.

The Wikipedia page for [Graphite](https://en.wikipedia.org/wiki/Graphite) gives 2.09–2.23 for density (maybe from the [Handbook of Mineralogy](http://www.handbookofmineralogy.com/pdfs/graphite.pdf)) and 1.9-2.3 for specific gravity (source unknown).

To be consistent with LBL, which is what the periodic table uses for xray data, I will change it to 2.2.

**Comment by pkienzle** [comment]
Fixed in 993538b96ba508e4f054f36714eadf76ead913d4.  This will appear in the next release (V1.5.2).
